bigbluebutton-html5/private/locales/en.json

```diff
@@ -172,18 +172,18 @@
     "app.actionsBar.emojiMenu.statusTriggerLabel": "Status",
     "app.actionsBar.emojiMenu.awayLabel": "Away",
     "app.actionsBar.emojiMenu.awayDesc": "Change your status to away",
-    "app.actionsBar.emojiMenu.raiseLabel": "Raise",
-    "app.actionsBar.emojiMenu.raiseDesc": "Raise your hand to ask a question",
-    "app.actionsBar.emojiMenu.undecidedLabel": "Undecided",
-    "app.actionsBar.emojiMenu.undecidedDesc": "Change your status to undecided",
+    "app.actionsBar.emojiMenu.raiseHandLabel": "Raise",
+    "app.actionsBar.emojiMenu.raiseHandDesc": "Raise your hand to ask a question",
+    "app.actionsBar.emojiMenu.neutralLabel": "Undecided",
+    "app.actionsBar.emojiMenu.neutralDesc": "Change your status to undecided",
     "app.actionsBar.emojiMenu.confusedLabel": "Confused",
     "app.actionsBar.emojiMenu.confusedDesc": "Change your status to confused",
     "app.actionsBar.emojiMenu.sadLabel": "Sad",
     "app.actionsBar.emojiMenu.sadDesc": "Change your status to sad",
     "app.actionsBar.emojiMenu.happyLabel": "Happy",
     "app.actionsBar.emojiMenu.happyDesc": "Change your status to happy",
-    "app.actionsBar.emojiMenu.clearLabel": "Clear",
-    "app.actionsBar.emojiMenu.clearDesc": "Clear your status",
+    "app.actionsBar.emojiMenu.noneLabel": "Clear",
```

Comment:
What's wrong with `clear`? It's what it does, isn't it? Clears the status.
clearLabel and clearDesc works pretty well here.